src/colorizer/Dataset.ts

```diff
@@ -130,7 +132,12 @@ export default class Dataset {
     const url = this.resolveUrl(metadata.data);
     const featureType = this.parseFeatureType(metadata.type);
 
-    const source = await this.arrayLoader.load(url);
+    const source = await this.arrayLoader.load(
+      url,
+      FeatureDataType.F32,
+      metadata.min ?? undefined,
+      metadata.max ?? undefined
```

Comment:
Oops, bug that I missed!

Reply:
It would use the min/max calculated from the data instead of letting the metadata min/max override it. There were a few features in NucMorph that excluded outliers from the final min/max calculation to avoid blowing out the range.